<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client</class>
 <widget class="QMainWindow" name="client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1084</width>
    <height>895</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Connect">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>251</width>
      <height>47</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IP">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>301</width>
      <height>44</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="Port">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>121</width>
      <height>44</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>8080</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>691</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="Status">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="Stable">
         <property name="styleSheet">
          <string notr="true">font: 75 20pt \&quot;Consolas\&quot;; color: #1d8aff;</string>
         </property>
         <property name="text">
          <string>Stable</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="CriStable">
         <property name="styleSheet">
          <string notr="true">font: 75 20pt &quot;Consolas&quot;;</string>
         </property>
         <property name="text">
          <string>CriStable</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="UnStable">
         <property name="styleSheet">
          <string notr="true">font: 75 20pt &quot;Consolas&quot;;</string>
         </property>
         <property name="text">
          <string>UnStable</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>1041</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="Plot">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCustomPlot" name="Voltage" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 85, 255);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCustomPlot" name="Current" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 170, 255);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCustomPlot" name="Frequency" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 170, 255);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCustomPlot" name="Power" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 255, 255);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>691</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="EngineState">
     <item>
      <layout class="QHBoxLayout" name="Engine_label">
       <item>
        <widget class="QLabel" name="Engine1_label">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(79, 170, 255);
color: rgb(237, 255, 255);
font: 75 16pt &quot;Candara&quot;;</string>
         </property>
         <property name="text">
          <string>Generator1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Engine2_label">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(79, 170, 255);
color: rgb(237, 255, 255);
font: 75 16pt &quot;Candara&quot;;</string>
         </property>
         <property name="text">
          <string>Generator2</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="EngineSet">
       <item>
        <widget class="QTextBrowser" name="Engine1">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Consolas&quot;;
border-color: rgb(79, 170, 255);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="Engine2">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Consolas&quot;;
border-color: rgb(79, 170, 255);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Alarm">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>160</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 0, 0);
font: 20pt &quot;Candara&quot;;</string>
    </property>
    <property name="text">
     <string>Alarm</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1084</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
